designing-your-journal/en/inclusive-and-accessible-theming

```diff
@@ -0,0 +1,48 @@
+
+Most OJS 3 themes allow you to customize the main or header colour of your website. With [WebAim’s Contrast Checker(https://webaim.org/resources/contrastchecker/), for example, you can check whether you should use black or white text, as well as header logo, with your main colour of choice in order to ensure readability for users with visual limitations. In the following example, the user is testing out a pink as well as a purple selected to match their homepage cover image. 
+
+![](./assets/designing-your-journal-inclusive-and-accessible-theming-WCAG-coulour-contrast-checker.png)
+
+The test recommends that the user only display black text on pink or white text on purple. Therefore, the user has two accessible options using these colours:
+
+![](./assets/designing-your-journal-inclusive-and-accessible-theming-header-colour-dont-1.png)
+![](./assets/designing-your-journal-inclusive-and-accessible-theming-header-colour-dont-2.png)
+![](./assets/designing-your-journal-inclusive-and-accessible-theming-header-colour-do-1.png)
+![](./assets/designing-your-journal-inclusive-and-accessible-theming-header-colour-do-2.png)
+
+Additionally, with tools such as the [Spectrum Chrome extension](https://chrome.google.com/webstore/detail/spectrum/ofclemegkcmilinpcimpjkfhjfgmhieb?hl=en) you can emulate various types of colour vision deficiencies (CVD) on any website. Tools like the [Accessible Color Palette Builder](https://toolness.github.io/accessible-color-matrix/) can help you decide on the best colour combinations for your website.
+
+Text colour should not be the only way of conveying important information, since it may not be available to a person with CVD and will be unavailable to screen reader users. Consider other visual emphasis (bold, italic) OR an explicit mention (in plain text) that the following information is important.
+
+### Alternative text for images
+You should provide ‘alt’ (alternative) text for images unless they are decorative or the text is present elsewhere on the page. Having such text is especially helpful for people with visual impairments who rely on a screen reader to have the content of the website read to them.
+
+Alt text can be added in designated fields or in the HTML.
+
+`<img src="learning-ojs3.1-jm-settings-workflow-sublib1.png" alt="submission library textbox">`
+
+Make sure to use descriptive language (e.g., describing a photograph in detail: "a woman in a lab coat holding a petri dish in a bright laboratory"). Avoid using "image of...” at the start of your description.
+
+### Descriptive URLs
+When writing URL text, make sure it can be understood without additional context. For example, use “see Author Guidelines” rather than “click here.”
+
+### Page structure
+When writing web content, ensure that:
+* Proper headings (e.g., <h1>, <h6>) are used to separate content in longer text. The headings should follow in order (e.g. <h1> - <h2> - <h3>) to allow a screen reader to navigate correctly through the page
```

Comment:
This might do with some clarification. A common misconception is that headings should always go in order and not be repeated:

```
<h1>1. XXXX</h1>
<h2>2. XXXX</h2>
<h3>3. XXXX</h3>
```

But `<h1>`, `<h2>`, and `<h3>` are actually _levels_ of headings:

```
<h1>Title</h1>
<h2>1. XXXX</h2>
<h2>2. XXXX</h2>
<h3>2.1 XXXX</h3>
<h3>2.2 XXXX</h3>
<h2>3. XXXX</h2>
```

Headings of the _same level_ should have the _same tag_.

When we we say the headings should "follow in order", what we mean is that headings shouldn't be skipped. The following is _invalid_ because a `<h3>` follows directly after a `<h1>`, without a `<h2>` between them:

```
<h1>Title</h1>
<h3>1 XXXX</h3>
<h2>2. XXXX</h2>
```

Finally, there should only ever be one `<h1>` on a page. Usually, the theme provides the `<h1>` and content creators should not create this themselves.